### PR TITLE
[MAINT] Remove copyright date from footer

### PR DESCRIPTION
**Pull request by Remi-Gau**
closes none

just update the copyright date in the footer... a few months before the end of the year...

### CODE REVIEW

**Comment by effigies** [comment]
Why not just drop the date? It has no legal status.

**Comment by effigies** [comment]
Just a reference (with a list of further references): https://hynek.me/til/copyright-years/

**Review by effigies — APPROVED** [review]

File: src/appendices/coordinate-systems.md [hunk 1]
```diff
@@ -212,7 +212,7 @@ Unless specified explicitly in the sidecar file in the
...
+| Talairach                          | Piecewise linear scaling of the brain is implemented as described in TT88.                                                                                                                                                                                                                                                                                                                                                                                                                                             |                          | [https://talairach.org/](https://talairach.org/)                                                                                     |
```
Comment (effigies) [inline src/appendices/coordinate-systems.md:215]:
```suggestion
| Talairach                          | Piecewise linear scaling of the brain is implemented as described in TT88.                                                                                                                                                                                                                                                                                                                                                                                                                                             |                          | [https://talairach.org/](https://talairach.org/)                                                                                   |
```

**Comment by Remi-Gau** [comment]
Ok I won't fix the links failing in this PR: at least now there is no error in CI only warning.

I think this deserves its own issue + PR so that we know it is a known issue other PRs can ignore.

**Pull request merged**